lte/gateway/c/session_manager/SessionStateEnforcer.h

```diff
@@ -159,6 +160,24 @@ class SessionStateEnforcer {
       std::unique_ptr<SessionState>& session, SessionFsmState target_state,
       SessionStateUpdateCriteria* session_uc);
 
+  /*
+   * Acquire and update session rules based on the IMSI
+   *
+   * */
+  uint32_t update_session_rules(
+      const std::string& imsi, std::unique_ptr<SessionState>& session_state,
+      bool gnb_teid_get, bool upf_teid_get,
+      SessionStateUpdateCriteria* session_uc);
+
+  /* update the GNB endpoint details in a rule */
+  bool insert_pdr_from_core(
+      const std::string& imsi, std::unique_ptr<SessionState>& session_state,

```

Comment:
Also, the naming that we often use for `std::unique_ptr<SessionState>&` is `session`

Reply:
followed the naming convention for std::unique_ptr<SessionState>& and replaced session_state with "session" in few places wherever possible for this PR